bin/get-jq

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
The `jq` comes with the Semaphore platform is `1.3` which doesn't support the `-e` flag.  Also the `install-package` command doesn't cache the `jq` apt package so it will be slow than this script which makes use of Semaphore cache.